### PR TITLE
Documenting Noise Sim

### PR DESCRIPTION
**Pull request by SalahedeenIssa**
Fixes https://github.com/Infleqtion/server-superstaq/issues/2597

### CODE REVIEW

**Comment by paaige** [comment]
<p>Made some comments on the notebooks (I'm new to reviewNB so sorry if this is confusing)</p>

**Review by paaige — COMMENTED** [review]
I made some comments in reviewNB. need to figure out how to get them to show up here..

**Comment by SalahedeenIssa** [comment]
I can't see them :/

File: docs/source/optimizations/hilbert/hilbert_compile_css.ipynb [hunk 1]
```diff
@@ -151,9 +151,53 @@
      "name": "stdout",
```
Comment (vtomole) [inline docs/source/optimizations/hilbert/hilbert_compile_css.ipynb:151]: <p>Hello</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

**Comment by SalahedeenIssa** [comment]
@paaige I responded to your comments in the review

**Comment by paaige** [comment]
I left one comment (small nit) but other than that it looks good, I'll approve.

**Review by paaige — APPROVED** [review]

**Pull request merged**